docker-unified/entrypoint.sh

```diff
@@ -89,14 +88,15 @@ sed -i "s/autostart=true/autostart=${BW_ENABLE_NOTIFICATIONS}/" /etc/supervisor.
 sed -i "s/autostart=true/autostart=${BW_ENABLE_SCIM}/" /etc/supervisor.d/scim.ini
 sed -i "s/autostart=true/autostart=${BW_ENABLE_SSO}/" /etc/supervisor.d/sso.ini
 
-chown -R $UID:$GID \
+chown -R bitwarden:bitwarden \
```

Comment:
```suggestion
chown -R $PUID:$PGID \
```

Since the `addgroup` and `adduser` commands can fail if the `PUID` or `PGID` provided already exists in the image, we should use the values here instead of `bitwarden:bitwarden`.